src/plugins/Debug.ts

```diff
@@ -44,15 +60,21 @@ class Debug extends GenericPlugin {
 
   private insertMessage(baseMessage: string): void {
     const currentDate = new Date();
-
     const hour = padTime(currentDate.getHours());
     const minutes = padTime(currentDate.getMinutes());
     const milliseconds = padTime(currentDate.getMilliseconds());
-
     const message = `[EVENT] ${baseMessage} at ${hour}:${minutes}:${milliseconds}`;
 
     insertElement('p', { style: MESSAGE_STYLE }, this.debugOverlay, message);
   }
+
+  private insertMetadata(metadata: googletag.events.SlotRenderEndedEvent): void {
+    const message = Object.entries(metadata)
+      .map(([key, value]) => `${key} = ${value}`)
+      .join('<br />');
+
```

Comment:
If we decide we dont want the `slot: [object Object]` we can remove it in this map and join but it's just another line of code which i know we want to limit. Groups decision.

Reply:
I'd say let's leave it for now